src/main/java/com/adobe/target/edge/client/service/DefaultTargetService.java

```diff
@@ -192,4 +169,23 @@ private String getBestLocationHint(TargetDeliveryRequest deliveryRequest) {
     }
     return stickyLocationHint;
   }
+
+  private HttpResponse<DeliveryResponse> callDeliveryApi(TargetDeliveryRequest deliveryRequest) {
+    String url = clientConfig.getUrl(getBestLocationHint(deliveryRequest));
```

Comment:
These new methods are great.  Can you add the `telemetryService.getTelemetry()` and `telemetryService. addTelemetry()` calls to both methods?  We want to be sure to collect and also send telemetry on the notifications requests.

Reply:
https://github.com/adobe/target-java-sdk/blob/ee646c170e08b5e2f5e9cf8c4de5c1e441fe9ff2/src/main/java/com/adobe/target/edge/client/ondevice/OnDeviceDecisioningService.java#L200

Telemetry is already sent for ODD as part of notification.
Is there something I am missing as part of flow.  I haven't made changes to ODD.
As we have added telemetry features, ODD should automatically get as part of **createTelemetry** method call.